about page has a new card view for team page. for testing only

syam3526 opened the pull request:
Bug Fixes ONLY. NO NEW FEATURE ACCEPTED!

Description


Related Issue


Motivation and Context


How Has This Been Tested?


Screenshots (if appropriate):

varghesethomase commented on about.html:
```diff
@@ -0,0 +1,766 @@
+        <meta name="keywords" content="mozilla,mozillakerala,free,opensource,volnteer,webmaker,hosting">
+ 
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
I hope it would be good to load the script tags at the end of the page as it effects the time for the initial render.

varghesethomase commented on contributors.html:
```diff
@@ -723,6 +690,36 @@ <h3 class="g-font-size-18--xs g-color--white">mozilla kerala</h3>
 })();
 </script>
+<script>
+    window.fbAsyncInit = function() {
```
Should we really track the page views in fb? @riverspirit @shinenelson any thoughts?

syam3526 replied:
will remove the tracking code

syam3526 replied:
removed the tracking code.

varghesethomase commented:
Also it would be good if you can align the html content properly so that its more readable. Another good suggestion would be to use 2 spaces instead of 4 as another way to improve readability.

syam3526 commented on CONTRIBUTING.md:
```diff
@@ -8,6 +8,9 @@ Rigin Ommen
 
 Franklin Antony
 
+Saurav Nair
```
I didn't create this file.  but will edit it though

riverspirit replied:
CONTRIBUTING.md is intended to hold information on 'how to contribute', not to provide names of contributors. See an example [here](https://github.com/puppetlabs/puppet/blob/master/CONTRIBUTING.md). So I suggest removing all the names from this file.

(Also, apart from that, I don't want myself mentioned anywhere in the site or code; just a personal preference :)

varghesethomase replied:
I agree with @riverspirit . Sorry that I never noticed the name of the file. Contributing.md would be for ways on how to contribute to a project.

syam3526 commented:
we can portrait every contributor we ever had as a token of appreciation.  that's why I left it as placeholders so we can add the right people. .
The founder's page is a recognition to the founders who founded mozillakerala , the contributor's page will list all the active contributors we have now.. which also include the founders.

syam3526 commented:
edited all the placeholders.   changed fonts to Zilla slab.

syam3526 commented:
edited the nav bar

riverspirit commented:
> The founder's page is a recognition to the founders who founded mozillakerala

That is already happening in the contributors page. @varghesethomase, what do you think?

syam3526 commented:
So we only need the contributors page and the founder page is not needed..
So I will put up a search panel. Since the list of contributors will be
huge.
.
Please accept the review and merge then delete the founders page.


On Wed, May 2, 2018, 10:19 PM jsx <notifications@github.com> wrote:

> The founder's page is a recognition to the founders who founded
> mozillakerala
>
> That is already happening in the contributors page. @varghesethomase
> <https://github.com/varghesethomase>, what do you think?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MozillaKerala/Home/pull/22#issuecomment-386044767>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AK6QdnNzRlgyDx8TJJMsTfnGq14xeycFks5tueOBgaJpZM4TvTOg>
> .
>

riverspirit commented:
Regarding the JavaScript search feature: This is an opportunity to recruit
more developers to this project. Make this feature into a GitHub issue and
post in the Telegraph group and someone will probably pick it up.

I'm not sure what you currently have in mind, but IMO, the contributors
page should ideally be populated from a static json file having the list of
all contributors. This will make implementing the search easier too.

Can you create a GitHub issue and post the link?

On Thu 3 May, 2018, 9:54 AM syamkumar, <notifications@github.com> wrote:

> added a simple search bar, removed the founder's page.
> need to work on the search javascript a little more. could use the help
> from someone on that.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MozillaKerala/Home/pull/22#issuecomment-386188253>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AArgzmD83hRXUikYYDkQ0CxBGApQ7fOtks5tuoZigaJpZM4TvTOg>
> .
>

syam3526 commented:
updated contributors.html and removed the buttons.   
need suggestions on the cards. should w do it in a single page or have multiple pages with 12 or 16 cards per page
